dashboard/app/controllers/section_libraries_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SectionLibrariesController < ApplicationController
```

Comment:
I think we'll want to leverage `authorize_resource`. Have you already set it up so students have access to projects from their sections.  AFAIK only teachers have that ability now, see: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/ability.rb

Reply:
Good call. I've updated it to use this authorization rule: https://github.com/code-dot-org/code-dot-org/blob/843d6b04f72a70e8560b62b25e9898f26a1adf6b/dashboard/app/models/ability.rb#L84